dataflow/src/main/java/org/checkerframework/dataflow/analysis/UnusedAbsValue.java

```diff
@@ -0,0 +1,17 @@
+package org.checkerframework.dataflow.analysis;
+
+import org.checkerframework.javacutil.BugInCF;
+
+/**
+ * UnusedAbsValue is an AbstractValue but is not involved in any computation during dataflow
+ * analysis. For those analyses which have no computation between two AbstractValues, it is
+ * sufficient to use UnusedAbsValue and unnecessary to implement another specific AbstractValue.
+ * Example analysis using UnusedAbsValue is LiveVariable analysis.
```

Comment:
You could mention issue #200 here and that this is a work-around for that issue.